woocommerce/woocommerce-android
# [Coupons] M1 Empty Screen

Part of: #5528

⚠️ Please do not merge yet as this is a continuation work from PR #6059  ⚠️ 



### Description


This PR adds the M1 Empty Screen for Coupons List:

<img width="239" alt="Screen Shot 2022-03-22 at 14 50 03" src="https://user-images.githubusercontent.com/266376/159432851-d8bc982f-9a88-4b50-97a8-b89be135eb11.png">


### Testing instructions


To test easily, go to `CouponsList.kt` and replace:

```
fun CouponListContainer(state: CouponListState) {
    when {
        state.coupons.isNotEmpty() -> CouponList(
            coupons = state.coupons,
            currencyCode = state.currencyCode
        )
        state.coupons.isEmpty() -> EmptyCouponList()
    }
}
```

with

```
fun CouponListContainer(state: CouponListState) {
    EmptyCouponList()
}
```

Then run the app, go to more Menu, and select Coupons. Ensure that the empty screen look matches the design.


### Images/gif


| Light | Dark |
|-|-|
| ![Light version of Coupons List Empty State UI](https://user-images.githubusercontent.com/266376/159434416-548abeb6-ffbb-49d5-a0a1-6489ffced817.png) | ![Dark version of Coupons List Empty State UI](https://user-images.githubusercontent.com/266376/159434409-77201345-25d4-4191-9c72-0ac21a7afbd2.png) |
